app/javascript/services/whitelists.js

```diff
@@ -10,7 +11,7 @@ const SQL_QUERIES = {
   getRegionWhitelist:
     'SELECT polyname, SUM(area_extent_2000) as total_extent_2000, SUM(area_extent) as total_extent_2010, SUM(area_gain) as total_gain, SUM(year_data.area_loss) as total_loss FROM data WHERE thresh = 0 AND {location} GROUP BY polyname',
   getWaterBodiesWhitelist:
-    "SELECT iso, adm1, adm2 from water_bodies_gadm36 WHERE iso = '{country}' AND adm1 = {region}"
+    "SELECT iso, adm1, adm2 from water_bodies_gadm36 WHERE iso = '{country}' AND gid_1 = '{region}'"
```

Comment:
Surely this will break if we are selecting adm1 and adm2? Is gid_1 the new gadm36 nomenclature?